test/stream_queue_test.dart

```diff
@@ -1046,6 +1046,9 @@ void main() {
       expect(events.withTransaction(expectAsync1((queue) async {
         expect(await queue.next, 2);
         throw 'oh no';
+        // TODO: Remove after https://github.com/dart-lang/sdk/issues/41156
```

Comment:
Can you mention this in the commit message too? That will make it link in github.